src/govuk/components/footer/footer.yaml

```diff
@@ -6,12 +6,15 @@ params:
   params:
   - name: visuallyHiddenTitle
     type: string
+    required: false
     description: Title for a meta item section, which defaults to Support links
   - name: html
     type: string
+    required: true
```

Comment:
I don't think this is true for `meta.html` or `meta.text` – this is only used if you want to add extra HTML or text near the links. If you don't provide either of them we just omit that section entirely:

https://github.com/alphagov/govuk-frontend/blob/abe1672b68fe08a55fb3562a684b408a5d452bb0/src/govuk/components/footer/template.njk#L47-L51

Reply:
@36degrees Thanks, I think I confused it with the `meta.items.text`! Have updated `meta.text` and `meta.html` to be `required: false`